deltachat-jsonrpc/src/api/types/account.rs

```diff
@@ -17,29 +17,43 @@ pub enum Account {
         // size: u32,
         profile_image: Option<String>, // TODO: This needs to be converted to work with blob http server.
         color: String,
+
+        /// Account IO is disabled when this account is not selected
+        ///
+        /// this means IO is stopped unless this account is selected
+        /// and background fetch is also disabled for this account
+        background_io_disabled: bool,
```

Comment:
maybe we should just store it inside of the accounts because we need to avoid editing `accounts.toml`, see https://github.com/deltachat/deltachat-desktop/issues/3671#issuecomment-2781389220